src/testUtils.js

```diff
@@ -0,0 +1,187 @@
+import react from 'react';
```

Comment:
We should not be introducing this file.  These utilities are in react-unit-test-utils